plugins/modules/source_control/gitlab/gitlab_deploy_key.py

```diff
@@ -145,6 +145,13 @@ def __init__(self, module, gitlab_instance):
     def createOrUpdateDeployKey(self, project, key_title, key_key, options):
         changed = False
 
+        # note: unfortunately public key cannot be updated directly by
+        #   gitlap rest api, so for that case we need to delete and
```

Comment:
```suggestion
        #   GitLab REST API, so for that case we need to delete and
```

Reply:
Ok, I added the changelog fragment and fixed the comment. Let me know if I should do anything else.